runtime/common/src/crowdloan.rs

```diff
@@ -304,23 +304,23 @@ decl_module! {
 		/// slot. It will be withdrawable in two instances: the parachain becomes retired; or the
 		/// slot is unable to be purchased and the timeout expires.
 		#[weight = 0]
-		fn contribute(origin, #[compact] index: FundIndex, #[compact] value: BalanceOf<T>) {
+		fn contribute(origin, #[compact] index: FundIndex, #[compact] value: BalanceOf<T>, return_to: T::AccountId) {
```

Comment:
the parachain AccountId type may not be same as Polkadot. e.g. it could be H160 for evm chain.

Reply:
Look at how we approached the claims pallet in Polkadot. A user can sign a message with their polkadot account and then assign the recipient for the claim. I dont think it is up to our pallet here to understand all ways that downstream chains may use this data.

Reply:
Make it `Vec<u8>` is enough to represent all different kinds of account types and we don't need another claim process

Reply:
Im not yet convinced, but will keep it in mind. Should be easy enough to update if we need to, but like, feels wrong instinctually